templates/install-config.yaml.j2

```diff
@@ -33,10 +33,11 @@ networking:
   type: "{{ ocp_network_type }}"
 platform:
   openstack:
+    defaultMachinePlatform:
+      type: {{ ocp_worker_flavor }}
```

Comment:
If ocp_worker_flavor is replacement for computeFlavor, then we need a default value for ocp_worker_flavor in group_vars.

Reply:
Set the default to `m4.xlarge` which is what it appears it was set to before.